### DISCUSSION TITLE
Green Tiles and why the issue is not AutoOrtho

### DISCUSSION THREAD

**Discussion by dpdurst** [kubilus1/autoortho#284] [Show and tell]
Many, Many folks are having issues with the green tiles showing up during flights and asking what can be done in AutoOrtho that will resolve the issue.

**_The short answer is nothing, this is not an AutoOrtho issue!_**

It does not matter what speed internet connection you have (I have 1 GB) it does not matter what sim your flying (XP11 or XP12) or the processor type (Intel - AMD) or pretty much anything else on your end.

What this does have to do with is everything outside of your house from your ISP all the way to the server that is being called to stream the data from.

You will notice more green tiles with faster jets  than you will with SEL or even slower MEL aircraft, however you will get them at some point. Even I have got them on flights that maybe a week earlier didn't happen flying a DA42 through the desert area's.

I've seen them in California and Arizona while flying as well. I've really seen them outside of some flights of the NA region. Canada, South America to name a few. 

So what can you do to resolve this, again answer is not much. Your most likely flying a Jet or faster business jet and you may be outside the NA region which seems more prone to streaming issues and getting the data than the NA region does.

1. In your AutoOrtho configuration file (c:\users\<your username>) directory is a file called .autoortho and you can edit this in the section that has the max_wait time to maybe something like 2.0 or 3.0 this will give more time to possible have the server stream the info.

2. Its possible that  your getting data limited further upstream from your ISP at the server or something else,  you could try a VPN but if they are already onto the IP address, they may catch this as well and start limiting data.

Outside of these few changes, not much more you can do but live with it. Unfortunately its a side affect of accessing the data from servers in different regions.

**Comment by Jeff2122** [comment]
In outside view, I sometimes see green tiles then swing around to another direction and back and pop! The green tile is gone.

**Comment by d41k4n** [comment]
There is an edge case to this that has nothing to do with download speeds or how responsive the remote image provider is, namely when distant terrain patches look "out of place" (i.e. tiled discontinuities) but will eventually "auto-correct" as the aircraft gets closer to them. This is very much an AutoOrtho issue because it is related to the fact that AO derives every texture mipmap level directly from downloaded imagery at the corresponding ZL which can be inconsistent across different ZL's due to imagery coming from different sources and/or having been obtained with different surveys depending on how the imagery provider decides to structure its imagery layers. IMO this is a known design limitation and briefly explained in [this post](https://forums.x-plane.org/index.php?/forums/topic/259020-autoortho-streaming-ortho-imagery-for-x-plane-11/&page=35#comment-2586995). 

FYI, this is in contrast to orthophoto scenery created locally with Ortho4XP, where lower-level mipmaps are _always_ derived (downsampled)  from the full-res image and therefore won't suffer from such visual discontinuities.
